### DISCUSSION TITLE
ADFS configuration DEX for ArgoCD

### DISCUSSION THREAD

**Discussion by marshall89923** [dexidp/dex#3796] [General]
hello everyone, 
I have configured dex config and ADFS as mentioned here:

[https://github.com/dexidp/dex/issues/1309#issuecomment-428655923](https://github.com/dexidp/dex/issues/1309#issuecomment-428655923)

and the SAML is working correctly. However in Argo Workflows, there are pipelines, and those pipelines are marked with the user executing them. 
In these pipelines, I need both the username and email, for example, to perform Git operations.
Currently, the jobs are only marked with the email, and the username is missing.
How can I configure adfs and DEX Config to use username and password and not only email?

Thank you very much

**Comment by agilgur5** [comment]
I'm not entirely sure what OP is referring to exactly, but this discussion was verbatim duplicated in Argo Workflows and I attempted to answer there: https://github.com/argoproj/argo-workflows/discussions/13768#discussioncomment-11007591
